core/components/ui/form/counter/counter.tsx

```diff
@@ -11,13 +11,13 @@ import {
 import { cn } from '~/lib/utils';
 
 interface Props extends Omit<ComponentPropsWithRef<'input'>, 'onChange'> {
```

Comment:
Do we want to extend ComponentPropsWithRef here still?

Reply:
We're gonna keep extending for now. Good call out, tho!